src/components/VAllResultsGrid/VImageCellSquare.vue

```diff
@@ -13,12 +13,12 @@
     >
       <img
         ref="img"
-        class="w-full h-full object-cover rounded-sm"
+        class="w-full h-full object-cover rounded-sm bg-dark-charcoal-10 text-dark-charcoal-10"
         loading="lazy"
         :alt="image.title"
         :src="getImageUrl(image)"
-        :width="image.width"
-        :height="image.height"
```

Comment:
these properties aren't actually defined, so it's better to use a default with the correct aspect ratio, for now.

Reply:
Are these properties never defined and will never be used? Or could we have something like `"image.width ?? 250"`?

Reply:
I second @obulat's comment, that'll allow us to send the dimensions from the backend without needed a frontend change/deploy.

Reply:
> absolutely should have dimensions coming from the API.

Yes, but none of our provider scripts set these fields. This suggestion is well-taken though! I'll do it this way.

Reply:
In fact, the StockSnap and the WordPress Photos Directory scripts do assign these values, for the rest it is a matter of updating them.

https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/providers/provider_api_scripts/stocksnap.py#L141-L142

https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/providers/provider_api_scripts/wordpress.py#L259-L260

Reply:
That's great! Unfortunately those properties aren't exposed on the list view, just in the single result view:

https://api.openverse.engineering/v1/images/?q=dog&source=stocksnap